src/js/no-react-entry.js

```diff
@@ -23,6 +24,18 @@ if (wizardPages.has(location.pathname)) {
 }
 
 const store = createCommonStore();
-
 createLoginWidget(store);
-createPensionApplicationStatus(store);
+if (pensionPages.has(location.pathname)) {
+  createApplicationStatus(store, {
+    formId: '21P-527EZ',
+    applyText: 'Apply for Veterans Pension'
```

Comment:
@peggygannon should the Pension apply button also include the word "benefits".

Reply:
@cehsu Yes, thanks for checking. Yes, I would recommend: Apply for Veterans Pension Benefits.

Reply:
Thanks @peggygannon, I'll loop that into the next set of similar changes.